File: seeker/snippet/Student.java
//date: 2023-12-25T16:41:21Z
//url: https://api.github.com/gists/c3190719558d2b073090526830b3ed84
//owner: https://api.github.com/users/elifdev

package org.example;

public class Student extends User
{
    private String address;
    private String creditCard;
    private String phone;

    // bir öğrencinin aldığı dersleri array şeklinde mi göstermeliyiz

    public String getAddress() {
        return address;
    }

    public void setAddress(String address) {
        this.address = address;
    }

    public String getCreditCard() {
        return creditCard;
    }

    public void setCreditCard(String creditCard) {
        this.creditCard = creditCard;
    }

    public String getPhone() {
        return phone;
    }

    public void setPhone(String phone) {
        this.phone = phone;
    }
}

File: seeker/snippet/Student.py
#date: 2023-03-22T16:53:29Z
#url: https://api.github.com/gists/c69278775f19315dd1310216b222458e
#owner: https://api.github.com/users/ashtanyuk

import datetime

class Student():
  def __init__(self, name, dob, number):
    self.name = name
    self.birth_date = dob
    self.phone_number = number

  def age_calculator(self):
    current_date = datetime.datetime.now().date()
    student_birthdate = datetime.datetime.strptime(self.birth_date, "%m/%d/%y").date()

    delta = current_date - student_birthdate
    age = int(delta.days/365)
    return age